bpf/lib/policy_log.h

```diff
@@ -28,11 +28,23 @@ struct policy_verdict_notify {
 	__u32	pad1; // align with 64 bits
 };
 
+static __always_inline bool policy_verdict_filter_allow(__u32 filter, __u8 dir)
+{
+	// The two commented way don't work somehow on 1.19 kernel
+	// return ((filter & (1 << (dir-1))) > 0);
+	// return ((filter & dir) > 0);
+	return ((filter & (0x80000000 + dir)) > 0);
+}
+
 static __always_inline void
 send_policy_verdict_notify(struct __ctx_buff *ctx, __u32 remote_label, __u16 dst_port,
 			   __u8 proto, __u8 dir, __u8 is_ipv6, int verdict,
-			   __u8 match_type)
+			   __u8 match_type, __u32 filter)
```

Comment:
Since this argument is the same from all call sites (at least for now) and is a global (config) variable, maybe we could get rid of that argument and instead directly match aginst `POLICY_VERDICT_LOG_FILTER` in `policy_verdict_filter_allow`?

Reply:
How to do that? Including the header file (lxc_config.h ) will result in the build fail.  What is the alternative way?

Reply:
I would define `POLICY_VERDICT_LOG_FILTER` at the top of this file if it isn't already defined. That should silence build warnings. Files that actually use `send_policy_verdict_notify()` (i.e., `bpf_lxc.c` right now) need to include `lxc_config.h` before including this file; I think that's already the case.

Reply:
Thanks for the suggestion. Updated the patch.